cPP_DSC.adoc

```diff
@@ -356,58 +356,58 @@ One way to categorize keys is by the cryptographic functions they are allowed to
 
 For a DSC, a connection is established between a CApp and the services provided by the DSC when keys or services are requested. Each time the CApp establishes a connection to the DSC the CApp is authenticated to ensure the CApp has authorization to the requested keys. Since a CApp (acting as an agent of the user) may utilize their DSC keys multiple times, the establishment of individual connections for each use can be a resource constraint for the DSC as authorization methods using public keys tend to be resource intensive (i.e. uses a fair amount of internal memory and takes a long time).
 
-As an alternative to requiring authorization for each access to a key, the DSC could allow the user or owner of the key to open a session. With a session, the CApp would provide the authentication data for the first connection, then the DSC would maintain the session and authorization using a series of less resource-intensive challenges and responses. In some instances the DSC may still require additional authorization (such as an elevation of privileges) to access keys (or different, related keys). Such a protocol of challenges and responses may generate and use ephemeral authorization tokens, which would be one form of critical security parameter (CSP). The DSC may have to switch session contexts in and out of the DSC to external temporary storage, which necessitates the protection of these CSPs. Such a session context is one type of SDO.
+As an alternative to requiring authorization for each access to a key, the DSC could allow the user or owner of the key to open a session. With a session, the CApp would provide the authentication data for the first connection, then the DSC would maintain the session and authorization using a series of less resource-intensive challenges and responses. In some instances the DSC may still require additional authorization (such as an elevation of privileges) to access keys (or different, related keys). Such a protocol of challenges and responses may generate and use ephemeral authorization tokens, which would be one form of SDE. The DSC may have to switch session contexts in and out of the DSC to external temporary storage, which necessitates the protection of these SDEs. Such a session context is one type of SDO.
```

Comment:
The previous sections introduce terminology such as SDO and SDE, but this section mainly uses "keys" and also sometimes refers to "CSPs". As far as I can tell, SDEs and CSPs are synonyms. I think it would be more coherent to replace the mentions of CSPs with SDEs.

Reply:
We need to review the use of CSPs within the document. @slpotte will propose a suggestion on this. We have a definition of SDEs as being a CSP, but no definition of CSP.

Sensitive data is also used in FCS_CKM.6.2.

GP_ROT document should also be consulted for the CSP definition as we are using that definition for RoT straight from their reference.

Reply:
Change reverted for now

Reply:
#263